build.sbt

```diff
@@ -125,6 +125,7 @@ lazy val slick =
       extTarget("slick"),
       name := "Slick",
       description := "Scala Language-Integrated Connection Kit",
+      version := "3.3.4-SNAPSHOT",
       libraryDependencies ++= Dependencies.mainDependencies,
```

Comment:
`SNAPSHOT` release is a moving target and it is not recommended to use them in production as builds become non-repeatable. For example see this link for a discussion about it: https://stackoverflow.com/questions/39964624/what-are-the-consequences-of-always-using-maven-snapshots

We'd better use a concrete version.

Reply:
We have an alternative track to get this accepted by open source slick in their repo. But meanwhile, for us to be able to test it on our timelines, this is the quickest way. Also, slick library is not very often updated except for maintenance related commits so it should be easy to maintain.

Reply:
+1 to not using snapshots since they are mutable. Is it possible to generate a pre-release version like `3.3.4-comments-rc.1`?

Reply:
Definitely agree that we should switch to a release version when we're ready to have the Iterable app use our additions to this library. But this PR doesn't get us all the way to a usable state; we still need to figure out how to call the `withComments` method implicitly, so we don't have to manually update all Slick queries in the app code.

Let's make a dedicated "release" PR when we're ready to build a JAR that the app can use?

Reply:
Yeah that sounds good.